backend/tests/Feature/NotificationTest.php

```diff
@@ -0,0 +1,150 @@
+<?php
+declare(strict_types=1);
+
+namespace Tests\Feature;
+
+use App\Models\User;
+use App\Notifications\SubmissionCreation;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Support\Facades\Notification;
+use Tests\TestCase;
+
+class NotificationTest extends TestCase
+{
+    use RefreshDatabase;
+
+    /**
+     * @return void
+     */
+    public function testSubmissionCreationNotificationForAnIndividualUser()
+    {
+        $user = User::factory()->create();
+        $notification_data = [
+            'body' => 'A submission has been created.',
+            'action' => 'Visit CCR',
+            'url' => '/',
+            'submission_id' => 1000,
+        ];
+        $user->notify(new SubmissionCreation($notification_data));
+        $this->assertEquals(1, $user->notifications->count());
+        $this->assertEquals("App\Notifications\SubmissionCreation", $user->notifications->first()->type);
```

Comment:
Expand the model to include all of "notification_data" and assert that the data preserved in the db matches the data passed

Reply:
This has been addressed.